src/main/java/shop/service/impl/ProductServiceImpl.java

```diff
@@ -1,4 +1,42 @@
 package shop.service.impl;
 
-public class ProductServiceImpl {
+import shop.dao.ProductDAO;
+import shop.dao.impl.ProductDAOImpl;
+import shop.entity.Product;
+import shop.service.ProductService;
+
+import java.util.*;
+
+public class ProductServiceImpl implements ProductService {
+
+    private ProductDAO productDAO = new ProductDAOImpl();
+
+    @Override
+    public void getProductByName(String name) {
+       Product product = productDAO.findProductByName(name);
+        if (product!=null) System.out.println(product);
```

Comment:
should return Product from method or throw exception smth like ProductNotFoundException. U will catch this exception in menu. And print or product or some message that product not found